ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -21,9 +21,104 @@
         static NS = FormView.Constants.NS;
         static IS = "adaptiveFormContainer";
         static bemBlock = 'cmp-adaptiveform-container';
+        static hamburgerSupport = false;
+        static nestingSupport = 3;
+
         static selectors  = {
             self: "[data-" + this.NS + '-is="' + this.IS + '"]',
+            menu: `.${FormContainerV2.bemBlock}-menu`,
```

Comment:
We have three containers here: MenuButton, menuNavbar, and menuDialog. Can we create three objects and define their selectors inside each object? This will make it easier to understand the role of the variables. Like this
menuNavbar: {
  nextButton: '',
  previousButton: ''
}

Reply:
Done!